Sources/Internal/FrameProvider.swift

```diff
@@ -323,6 +346,7 @@ final class FrameProvider {
       (monthsLayout.pinDaysOfWeekToTop ? 0 : (daySize.height + content.verticalDayMargin)) +
       (CGFloat(numberOfRows) * daySize.height) +
       (CGFloat(numberOfRows - 1) * content.verticalDayMargin) +
+      monthFooterHeight +
```

Comment:
same as https://github.com/airbnb/HorizonCalendar/pull/75/files#r531198063

Reply:
Updated to include `monthDayInsets.bottom`